src/Composer/InlineComposer.vala

```diff
@@ -36,6 +36,9 @@ public class Mail.InlineComposer : Gtk.ListBoxRow {
         composer = new ComposerWidget.inline ();
         composer.margin_top = 6;
         composer.has_recipients = true;
+        if (prev_chain_message != null && prev_chain_message.get_subject () != null) {
+            composer.subject = _("Re: %s").printf (prev_chain_message.get_subject ());
```

Comment:
The [email RFC](https://datatracker.ietf.org/doc/html/rfc2822#section-3.6.5) makes this string verbatim (so don't translate it) also we have to make sure that the subject doesn't already contain it before re-adding it.